charts/fulcio/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
i'm good to have this

@sabre1041 does that is ok for you?

Reply:
Probably good to mention is that it is easy to manage this see #300 on how folks can update when bumping the dependency versions.